### ISSUE TITLE
v0.2.0 and Maven

### ISSUE THREAD

**Issue by jnweaver** [UWMadisonUcomm/uw-ui-toolkit#61]
Hey, @nblair - I just pushed a new release with @jhanstra's MyUW styleguide and some other style changes.

I'm not clear on what needs to happen as far as Maven and publishing this release to our public artifact repository. Is this something you need to do? Should I notify you every time we put out a new release? Is this something we should incorporate in our build/release process? (E.g. we use Grunt to publish a snapshot and latest release to AWS S3.)

I did update the version number in pom.xml and the Maven readme -- not sure if I should have even done that.

**Comment by nblair** [comment]
Actually, you shouldn't ever manually touch the version element; there is a plugin that does that automatically. I'll take it from here :)

**Comment by nblair** [comment]
I'll push a few docs regarding how to do the maven releases as well; it does require certain setup.

**Comment by nblair** [comment]
Release complete:

<dependency>
  <groupId>edu.wisc.uc</groupId>
  <artifactId>uw-ui-toolkit</artifactId>
  <version>0.2.0</version>
</dependency>

See README-Maven.md in ad08db4 for additional detail on the release process. Look ok?

**Comment by jnweaver** [comment]
Thanks, Nick. We just published 0.2.1 to fix some Bower ignore issues, but that won't affect Maven.

I've installed Maven. If I wanted to publish this to the UW public Maven repository I'd need those privileges. But if we're going to move towards publishing to Maven Central in the future, we don't need to worry about that. So I'll let you handle publishing releases to the UW Maven repo for now, okay?
